app/views/smart_proxies/welcome.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for(:title, _("Smart Proxies")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("sitemap", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Proxies') %></h1>
+  <p><%= _("The Smart Proxy is a project which provides a restful API to various sub-systems.
+           Its goal is to provide an API for a higher level orchestration tools (such as Foreman).
+           The Smart proxy provides an easy way to add or extended existing subsystems and APIs using plugins.") %></p>
```

Comment:
1. "Proxy", inconsistent with line above.
1. Not too useful to describe the plugin capability here, it should focus on why it's managed in Foreman here.